envoy/tests/legacy/common.py

```diff
@@ -1,11 +1,7 @@
 import os
```

Comment:
Since `legacy` variants are being removed, what does the `/legacy` test directory mean? Are these tests specifically for `api_v2`?